1. Известно, что генеральная совокупность распределена нормально со средним квадратическим отклонением, равным 16. Найти доверительный интервал для оценки математического ожидания a с надежностью 0.95, если выборочная средняя M = 80, а объем выборки n = 256.

In [1]:
sigma <- 16
gamma <- 0.95
m <- 80
n <- 256

    Сначала найдём обратное значение функции стандартного нормального распределения.

In [2]:
c <- qnorm((gamma + 1) / 2)
c

[1] 1.959964

    Найдем верхнюю границу доверительного интервала:

In [3]:
t_1 <- m + sigma * c / sqrt(n)
t_1

[1] 81.95996

    Найдём нижнюю границу доверительного интервала:

In [4]:
t_2 <- m - sigma * c / sqrt(n)
t_2

[1] 78.04004

    Значит доверительный интервал равен:

In [5]:
c(t_2, t_1)

[1] 78.04004 81.95996

2. В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью, получены опытные данные: 6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1 Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей, оценить истинное значение величины X при помощи доверительного интервала, покрывающего это значение с доверительной вероятностью 0,95.

    Создаём массив данных измерений и записываем в соответствующие значения необходимые показатели выборки:

In [6]:
x <- c(6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1)
m <- mean(x)
sigma <- sd(x)
n <- 10

    Так как доверительная вероятность совпадает с предыдущей задачей, то будем использовать существующую величину:

In [7]:
c

[1] 1.959964

    Найдем верхнюю границу доверительного интервала:

In [8]:
t_1 <- m + sigma * c / sqrt(n)
t_1

[1] 6.869405

    Найдём нижнюю границу доверительного интервала:

In [9]:
t_2 <- m - sigma * c / sqrt(n)
t_2

[1] 6.310595

    Значит с надежностью в 0.95 мы можем сказать, что величина Х находится в данном интервале:

In [10]:
c(t_2, t_1)

[1] 6.310595 6.869405

3. Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм. Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр оказался равным 17.5 мм, а дисперсия известна и равна 4 кв.мм.

In [11]:
m_0 <- 17
m <- 17.5
n <- 100
sigma <- 4
alpha <- 0.05

    Наблюдаемое значение критерия равно:

In [12]:
f <- (m - m_0) * sqrt(n) / sigma
f

[1] 1.25

    При этом критическую точку для данного одностороннего критерия найдём как обратное значение функции стандартного нормального распределения от вероятности принятия правильной гипотезы:

In [13]:
c <- qnorm(1 - alpha)
c

[1] 1.644854

    Так как критическая точка выше, чем наблюдаемое значение критерия (1.25 < с), то можем утверждать, что можем принять существующую нулевую гипотезу.

4. Продавец утверждает, что средний вес пачки печенья составляет 200 г. Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет: 202, 203, 199, 197, 195, 201, 200, 204, 194, 190. Известно, что их веса распределены нормально. Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?

    Создаём массив данных измерений и записываем в соответствующие значения необходимые показатели выборки:

In [14]:
x <- c(202, 203, 199, 197, 195, 201, 200, 204, 194, 190)
m_0 <- 200
m <- mean(x)
sigma <- sd(x)
n <- 10
gamma <- 0.99

    Наблюдаемое значение критерия равно:

In [15]:
f <- (m - m_0) * sqrt(n) / sigma
f

[1] -1.065107

    В данном случае будем использовать таблицу критических значений t-критерия Стьюдента. Значение по таблице с количеством степеней свободы 9 и уровнем значимости 0.01 равно 3.25. И так как наблюдаемое значение критерия меньше критического значения (1.065 < 3.25), то можем принять существующую нулевую гипотезу.